### PR TITLE
docs: Make badges consistent, run markdownlint on all .md files

### PR DESCRIPTION
**Pull request by spetrosi**
- Consistently generate badges for GH workflows in README RHELPLAN-146921
- Run markdownlint on all .md files
- Add custom-woke-action if not used already
- Rename woke action to Woke for a pretty badge

Signed-off-by: Sergei Petrosian <spetrosi@redhat.com>

### CODE REVIEW

**Review by Jakuje — COMMENTED** [review]
should we fix also the ansible lint issues or do you want to submit fix in separate PR?

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
....yml/badge.svg)](https://github.com/linux-system-roles/ssh/actions/workflows/ansible-debian.yml) [![ansible-lint.yml](https://github.com/linux-system-roles/ssh/actions/workflows/ansible-lint.yml/badge.svg)](https://github.com/linux-system-roles/ssh/actions/workflows/ansible-lint.yml) [![ansible-test.yml](https://github.com/linux-system-roles/ssh/actions/workflows/ansible-test.yml/badge.svg)](https://github.com/linux-system-roles/ssh/actions/workflows/ansible-test.yml) [![ansible-ubuntu.yml](https://github.com/linux-system-roles/ssh/actions/workflows/ansible-ubuntu.yml/badge.svg)](https://github.com/linux-system-roles/ssh/actions/workflows/ansible-ubuntu.yml) [![shellcheck.yml](https://github.com/linux-system-roles/ssh/actions/workflows/shellcheck.yml/badge.svg)](https://github.com/linux-system-roles/ssh/actions/workflows/shellcheck.yml) [![markdownlint.yml](https://github.com/linux-system-roles/ssh/actions/workflows/markdownlint.yml/badge.svg)](https://github.com/linux-system-roles/ssh/actions/workflows/markdownlint.yml) [![woke.yml](https://github.com/linux-system-roles/ssh/actions/workflows/woke.yml/badge.svg)](https://github.com/linux-system-roles/ssh/actions/workflows/woke.yml)
```
[the remark this answers is not in the document]
Comment (spetrosi) [reply README.md:3]: The advantage is to generate them in .github, see https://github.com/linux-system-roles/.github/pull/42/files#diff-99e99bac7991cbf3337188c9f34cc45e998346d41ab9467aeea9406ad2654572R163
Doing this multi-line will be much harder. And given that there is automation, no need to maintain this line manually.

[the remark this answers is not in the document]
Comment (Jakuje) [reply README.md:3]: Thanks for clarification! It was not sure to me on the first couple of reads through the changes here.

[the remark this answers is not in the document]
Comment (spetrosi) [reply README.md:3]: It is nasty, yeah. There are spaces between badges to distinguish between them. Automation helps very much here.

**Pull request merged**